### DISCUSSION TITLE
Animate PrimeReact Tree Expanding/Collapsing?

### DISCUSSION THREAD

**Discussion by JDGatGitHub** [primefaces/community#446] [PrimeReact]
Is there a property to animate the expanding and collapsing of tree leaves?  Any other way?

**Comment by mertsincan** [comment]
@gucal @nitrogenous could you please check it? I think you can do it with custom css.

**Comment by nitrogenous** [comment]
Hey @JDGatGitHub I hope your are doing fine! I am going to suggest you a solution until we discuss about implementing this feature and I will also let you know whenever we decide about it! I am aware that is not the best solution but you can mix our PT options with PrimeFlex library to get what you want for now. 

https://primeflex.org/animations

https://github.com/primefaces/primereact/issues/5814

```
<Tree
    value={nodes}
    className="w-full md:w-30rem"
    pt={{
        subgroup: ({ context }) => ({
            className: context.expanded ? 'fadeinup' : 'fadeoutdown'
        })
    }}
/>;
```

**Reply by JDGatGitHub** [reply]
Thanks @nitrogenous for the suggestion.  I implemented that but the subgroups/branches are animated visible from the top of the tree component behind other branches to the expanded area.  I see you create 5814 so probably just will wait for that

**Reply by glinkot** [reply]
Hi there.  I'm trying to apply the above passthrough, but it seems the 'subGroup' property isn't exposed:

"Object literal may only specify known properties, and 'subgroup' does not exist in type 'TreeTablePassThroughOptions'."

I'm figuring we can use primeflex to do animations at various points - if there's an example of this with the treeTable I'd really appreciate a link!
